### ISSUE TITLE
Please elaborate on your documentation with an example on how to call the hashed generated file from HTML

### ISSUE THREAD

**Issue by lmuzquiz** [Dragory/gulp-hash#30]
Hello, i read your complete documentation on the plugin. And i got half of it working, since i get the hashed names. The only problem is that from reading your documentation i can't understand how to call the hashed file from my HTML.  Could you elaborate an example on how to call the file on your documentation please?

Do i have to put <%= name %>-<%= hash %><%= ext %> on my HTML and that will convert to the name.hashed.extention or what? cause i already tried and its not working.

**Comment by Dragory** [comment]
No, this extension does not touch the HTML. You'll need to use a second plugin such as https://github.com/Dragory/gulp-hash-references for that.
